Installer/Installer.cs

```diff
@@ -22,7 +22,7 @@
     Platform = Platform.x64,
     Description = "The RevitPythonShell adds an IronPython interpreter to Autodesk Revit and Vasari.",
     UI = WUI.WixUI_InstallDir,
-    Version = new Version(),
+    Version = new Version(version),
```

Comment:
ah! there we go! ignore previous comment on this...

Reply:
yes, i has resolved this.